lintro/tools/definitions/vue_tsc.py

```
@@ -451,168 +435,313 @@ def check(self, paths: list[str], options: dict[str, object]) -> ToolResult:
+            for temp in temp_files:
+                try:
+                    if temp.exists():
+                        temp.unlink()
+                except OSError:
+                    pass
+
+    # -----------------------------------------------------------------
+    # Shared vue-tsc execution + output parsing
+    # -----------------------------------------------------------------
+
+    def _run_vue_tsc_and_parse(
```

Ruff (0.15.9)

[warning] Too many return statements (8 > 6)

(PLR0911)